script.js

```diff
@@ -1,4 +1,185 @@
+// Global variables
+var player = [];
+var computer = [];
+
+//make an empty deck with a current suit
+var makeDeck = function () {
+  var cardDeck = [];
+  var suits = ["Hearts", "Diamonds", "Clubs", "Spades"];
+  //create index for suits and ranks
+  var suitIndex = 0;
+  while (suitIndex < suits.length) {
+    var currentSuit = suits[suitIndex];
+    var rankIndex = 1;
+    while (rankIndex <= 13) {
+      var cardName = rankIndex;
+      if (rankIndex == 1) {
+        cardName = "Ace";
+      } else if (rankIndex == 11) {
+        cardName = "Jack";
+      } else if (rankIndex == 12) {
+        cardName = "Queen";
+      } else if (rankIndex == 13) {
+        cardName = "King";
+      }
+
+      //add values for cards
+      var cardValue = rankIndex;
+      if (cardValue == 1) {
+        cardValue = 1;
+      } else if (cardValue == 11 || cardValue == 12 || cardValue == 13) {
+        cardValue = 10;
+      }
+
+      // create object for card
+      var card = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankIndex,
+        value: cardValue,
+      };
+      //push the card to the deck
+      cardDeck.push(card);
+      rankIndex += 1;
+    }
+    suitIndex += 1;
+  }
+
+  return cardDeck;
+};
+
+//make random index for shuffling cards (mathfloor/random)
+var getRandomIndex = function (max) {
+  return Math.floor(Math.random() * max);
+};
+
+//Shuffle cards function
+cardDeck = makeDeck();
+var shuffleCards = function (cardDeck) {
+  var index = 0;
+  while (index < cardDeck.length) {
+    var randomIndex = getRandomIndex(cardDeck.length);
+    var randomCard = cardDeck[randomIndex];
+    var currentCard = cardDeck[index];
+    cardDeck[index] = randomCard;
+    cardDeck[randomIndex] = currentCard;
+    index = index + 1;
+  }
+  return cardDeck;
+};
+var shuffledDeck = shuffleCards(cardDeck);
+
+var gameMode = "deal cards";
+var sumOfComputer = "";
+var sumOfPlayer = "";
+var playerCard1 = "";
+var playerCard1 = "";
```

Comment:
Did you mean playerCard1 and playerCard2 ? 

Would be better to use an array for storing cards. so you don't have to hard code a new var everytime the player chooses to hit 

For example 
playerCards = [];

// add card to player 
playerCards.push( shuffledDeck.pop() );